platform/mellanox/sdk-src/sys-sdk/Makefile

```diff
@@ -1,24 +1,29 @@
 .ONESHELL:
 SHELL = /bin/bash
 
-MAIN_TARGET = sx-hash-calc_1.mlnx.$(MLNX_SDK_DEB_VERSION)_amd64.deb
-DERIVED_TARGETS = sx-hash-calc-dbgsym_1.mlnx.$(MLNX_SDK_DEB_VERSION)_amd64.deb
-PACKAGE_NAME = sx_hash_calc
+MAIN_TARGET = sys-sdk_1.mlnx.$(MLNX_SDK_DEB_VERSION)_$(CONFIGURED_ARCH).deb
+DERIVED_TARGETS = 
+PACKAGE_NAME = sys_sdk
 
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+
 	# get sources
-	rm -rf $(PACKAGE_NAME)-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION)
+
+	rm -rf sys_sdk-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION)
 
 	wget -c $(MLNX_SDK_SOURCE_BASE_URL)/$(PACKAGE_NAME)-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION).tar.gz -O - | tar -xz
 
 	# build
+
 	pushd $(PACKAGE_NAME)-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION)
 
-	if [ -f autogen.sh ]; then
-		./autogen.sh
-	fi
+	cmake -B builds/cmake -DSKIP_UNINSTALL_INSTALL_DEPENDENCY=on -DSKIP_DEPMOD_IN_INSTALL_PHASE=on -DUSE_API_TESTER=OFF -DCMAKE_PROJECT_VERSION=$(MLNX_SDK_DEB_VERSION)
```

Comment:
Does it works with SONIC_DEBUGGING_ON, SONIC_PROFILING_ON?
(https://github.com/sonic-net/sonic-buildimage/blob/master/rules/config#L95)

Reply:
Need to check. Thanks for this comment